# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/trunk/webrtc/modules/desktop_capture/linux/window_finder_x11.cc
# Commit: f9a9da6129c0
# Full Hash: f9a9da6129c01b5cefdc2ba2871d5bfe9fd0f0ab
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2019-04-17 21:47:29
# Regressor Bug: 1496359
# File Overlap Count: 1
# Description:
#   Bug 1496359 - Conflict resolutions for Pipewire support patch; r=ng
#   
#   Depends on D27368
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D27369
# ==============================================================================

diff -r 3f0981f37c1c -r f9a9da6129c0 media/webrtc/trunk/webrtc/modules/desktop_capture/linux/window_finder_x11.cc
--- a/media/webrtc/trunk/webrtc/modules/desktop_capture/linux/window_finder_x11.cc	Mon Apr 15 17:26:27 2019 +0000
+++ b/media/webrtc/trunk/webrtc/modules/desktop_capture/linux/window_finder_x11.cc	Mon Apr 15 17:31:02 2019 +0000
@@ -8,9 +8,11 @@
  *  be found in the AUTHORS file in the root of the source tree.
  */
 
-#include "modules/desktop_capture/window_finder_x11.h"
+#include "modules/desktop_capture/linux/window_finder_x11.h"
 
-#include "modules/desktop_capture/x11/window_list_utils.h"
+#include <memory>
+
+#include "modules/desktop_capture/linux/window_list_utils.h"
 #include "rtc_base/checks.h"
 #include "rtc_base/ptr_util.h"
 